doc/contributing.md

```diff
@@ -511,6 +514,93 @@ contains rules for running the linter checks:
 make check   # Runs ruff in check mode
 ```
 
+### Wrapping a GMT Module
+
+Wrapping a GMT module in PyGMT is usually a big task, but it can progress more smoothly
+and efficiently when divided into **small, manageable chunks**. This section gives an
+overview of the main tasks involved.
+
+1. Create a feature request ("wrapper request issue") for wrapping a GMT module and
+   discuss what users would like to see in the new feature. [optional, usually done by
+   users].
+2. Create a tracking issue ("wrapper tracking issue"), using the "Wrapper for a GMT
+   module" issue template, to track the progress of wrapping the module. Link it to the
+   Project board, and close the "wrapper request issue" with a comment such as
+   > Thank you for opening the feature request. The progress of wrapping the module will
+   > be tracked in issue #XXX and
+   > the [Project board](https://github.com/orgs/GenericMappingTools/projects/3).
+   [usually done by maintainers].
+3. Open one PR for the initial implementation, focusing on required and essential
+   parameters.
+4. Close the "wrapper tracking issue" once the initial implementation is merged. Leave a
+   comment such as:
+   > The initial implementation of wrapping the XXX module was completed in PR #XXX.
+   > Not all functionalities are implemented yet. Further progress will be tracked in
+   > the Project board.
+   This is necessary to avoid having hundreds of long-term open issues.
+5. Open one or more PRs to implement the remaining features and missing aliases.
+6. Open one PR to add a gallery example or a tutorial.
+
+These pull requests can be split among multiple contributors. There is no obligation for
+a single contributor to complete all steps. Please comment on the "wrapper tracking
+issue" if you would like to open a pull request for any of these tasks to avoid
+redundant efforts.
+
+#### Feature Request Issue for Wrapping a GMT Module
+
+* Find the [*Issues*](https://github.com/GenericMappingTools/pygmt/issues) tab at the
+  top of the GitHub repository and click *New Issue*.
+* Choose the *Feature request* issue template.
+* Follow the prompts for filling out the issue template.
+
+#### Central Issue for Tracking the Progress
+
+*This step is usually done by maintainers.*
+
+* Find the [*Issues*](https://github.com/GenericMappingTools/pygmt/issues) tab on the
+  top of the GitHub repository and click *New Issue*.
+* Choose the "Wrapper for a module" issue template.
+* Fill out the issue template.
```

Comment:
Maybe remove these two sections, since they're already covered in https://pygmt-dev--1687.org.readthedocs.build/en/1687/contributing.html#submitting-a-feature-request.